requirements-dev.txt

```diff
@@ -1,3 +1,5 @@
 -r requirements.txt
+requests==2.31.0
+pandas==2.1.4
```

Comment:
Unless there's a specific reason to I think it's better to not put version constraints on dev packages